src/common/micro/lima.F90

```diff
@@ -65,7 +65,7 @@ SUBROUTINE LIMA ( D, CST, ICED, ICEP, ELECD, ELECP, BUCONF, TBUDGETS, KBUDGETS,
                                 LFEEDBACKT, NMAXITER, XMRSTEP, XTSTEP_TS,               &
                                 LSEDC, LSEDI, XRTMIN, XCTMIN, LDEPOC, XVDEPOC,                  &
                                 NMOM_C, NMOM_R, NMOM_I, NMOM_S, NMOM_G, NMOM_H
-
+USE MODD_CH_AEROSOL,      ONLY: NSP,NCARB,NSOA
```

Comment:
We cannot use variables from a module in LIMA.
A (minimal) structure must be added to store these dimensions, or the dimensions should be added as dummy arguments